index.js

```diff
@@ -32,6 +35,7 @@ module.exports = {
         'promise/no-promise-in-callback': 'warn',
         'promise/no-callback-in-promise': 'warn',
         'promise/avoid-new': 'warn',
+        'promise/no-new-statics': 'warn',
```

Comment:
I added this rule to the recommended configuration as a warning - does this need adjustment?

Reply:
I'm thinking this should actually be an error. Playing around in a Node REPL:

```
❯ node
> new Promise.resolve()
TypeError: Promise.resolve is not a constructor
```